### PR TITLE
fix quo2 colors and button component

### PR DESCRIPTION
**Pull request by Parveshdhull**
### Summary

PR addresses following changes
- Design Team refactored the [foundations color file](https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Mobile-Foundations?node-id=619%3A10064)
	- neutral-100 is the new black
	- removed unused colors
- Button component
	- icon color is different as per position (above, before, or only icon)
	- missing some configurations 
		- dark-grey
		- photo background
		- blurred background
		- shell background (not in component libary, but used in shell))
- Some components are in the wrong directory
	- lowest_price
	- network_amount
	- network_breakdown
	- dropdown

status: ready

### CODE REVIEW

File: src/quo2/components/buttons/button.cljs [hunk 1]
```diff
@@ -6,70 +6,125 @@
...
-                                 :border-color {:default colors/neutral-30
-                                                :pressed colors/neutral-40}}
-                     :ghost     {:icon-color       colors/neutral-50
-                                 :label            {:style {:color colors/black}}
-                                 :background-color {:pr:pressedess colors/neutral-10}}
-                     :success   {:icon-color       colors/white
-                                 :label            {:style {:color colors/white}}
-                                 :background-color {:default colors/success-50
-                                                    :pressed colors/success-70
-                                                    :disabled colors/success-50}}
```
Comment (Parveshdhull) [inline src/quo2/components/buttons/button.cljs:35]: Figma buttons component doesn't have a success variant, and also I didn't find any use of this variant except in [activity_center](https://github.com/status-im/status-mobile/blob/develop/src/status_im/ui/screens/activity_center/views.cljs).
@ilmotta Please can you confirm if the success button is used for that component, and if used, please can you also provide the Figma link? Thank you

File: src/quo2/screens/main.cljs [hunk 2]
```diff
@@ -82,16 +82,7 @@
...
-          {:name      :lowest-price
-           :insets    {:top false}
-           :component lowest-price/preview-lowest-price}
-          {:name      :network-breakdown
-           :insets    {:top false}
-           :component network-breakdown/preview-network-breakdown}
-          {:name      :network-amount
-           :insets    {:top false}
-           :component network-amount/preview}]
+           :component information-box/preview-information-box}]
```
Comment (Parveshdhull) [inline src/quo2/screens/main.cljs:85]: @ibrkhalil Please don't put components in random directories. We use Figma like directory structure so that components are easy to find

File: src/quo2/components/counter/counter.cljs [hunk 3]
```diff
@@ -6,29 +6,28 @@
...
 
 (defn get-color [key]
   (get-in themes [(theme/get-theme) key]))
 
 (defn counter
-  "type:    default, secondary, grey
-   outline: true, false
+  "type:    default, secondary, grey, outline
    value:   integer"
-  [{:keys [type outline override-text-color override-bg-color style]} value]
```
Comment (J-Son89) [inline src/quo2/components/counter/counter.cljs:22]: why is this change here? 🤨

File: src/quo2/components/dropdowns/dropdown.cljs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-(ns quo2.components.list-items.dropdown
+(ns quo2.components.dropdowns.dropdown
```
Comment (J-Son89) [inline src/quo2/components/dropdowns/dropdown.cljs:1]: 👍

**Review by J-Son89 — APPROVED** [review]

File: src/quo2/components/wallet/lowest_price.cljs [hunk 5]
```diff
@@ -1,4 +1,4 @@
-(ns quo2.components.info.lowest-price
+(ns quo2.components.wallet.lowest-price
```
Comment (ibrkhalil) [inline src/quo2/components/wallet/lowest_price.cljs:1]: Thanks man

File: src/quo2/screens/main.cljs (hunk 2 above)
Comment (ibrkhalil) [reply src/quo2/screens/main.cljs:85]: Cool Parvesh, Thanks
I'll use Figma structure to put them in their correct directories

**Review by ibrkhalil — APPROVED** [review]
lgtm, Thanks for setting my components in the correct directory Parvesh!

**Review by flexsurfer — APPROVED** [review]
thank you
